boomkat_importer.user.js

```diff
@@ -1,23 +1,16 @@
 // ==UserScript==
 // @name        Import Boomkat releases to Musicbrainz
 // @description Add a button on Boomkat release pages to open MusicBrainz release editor with pre-filled data for the selected release
-// @version     2020.07.21.1
+// @version     2024.09.10.1
```

Comment:
When I wanted to install this userscript for testing, I realized that it is the only script for which the README has no install button. If you add the following two metadata lines, `tools/generate_README.py` will add one:

```suggestion
// @version     2024.09.10.1
// @downloadURL    https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/boomkat_importer.user.js
// @updateURL      https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/boomkat_importer.user.js
```